### PR TITLE
Add setter for KmlDataSource.name

### PR DESCRIPTION
**Pull request by burn123**
Related to #5656.
This allows the ability to set the name of a `KmlDataSource`.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ Change Log
 * Added ability to show tile urls in the 3D Tiles Inspector. [#5592](https://github.com/AnalyticalGraphicsInc/cesium/pull/5592)
 * Added behavior to `Cesium3DTilesInspector` that selects the first tileset hovered over if no tilest is specified. [#5139](https://github.com/AnalyticalGraphicsInc/cesium/issues/5139)
 * Added ability to provide a `width` and `height` to `scene.pick`. [#5602](https://github.com/AnalyticalGraphicsInc/cesium/pull/5602)
+* Added setter for `KkmlDataSource.name` to specify a name for the datasource.
```
Comment (hpinkos) [inline CHANGES.md]: Typo `Kkml`

File: Source/DataSources/KmlDataSource.js [hunk 2]
```diff
@@ -2344,14 +2344,20 @@ define([
...
          * This will be automatically be set to the KML document name on load.
          * @memberof KmlDataSource.prototype
          * @type {String}
          */
         name : {
             get : function() {
                 return this._name;
+            },
+            set : function(value) {
+                if(this._name !== value) {
```
Comment (hpinkos) [inline Source/DataSources/KmlDataSource.js]: Add a space after if: `if(this` -> `if (this`

**Comment by hpinkos** [comment]
THanks @burn123! I just had those two comments.  I'll let @mramato take a look at this when it's ready and merge

**Comment by burn123** [comment]
@hpinkos Can you trigger a rebuild? This should be ready

**Comment by hpinkos** [comment]
Thanks @burn123!  I just pushed a change to remove an extra line of whitespace, then I'll merge this as soon as CI passes.

**Comment by burn123** [comment]
@hpinkos I think there is something wrong with the Travis builds, because I keep getting timeout errors

**Comment by hpinkos** [comment]
No problem.  We pushed a few changes last week that fixed CI in master, but your branch might not be up to date.  I ran the test locally and everything is great.  Thanks!

**Pull request merged**